plugins/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/HuaweiCloudOrchestratorPlugin.java

```diff
@@ -61,7 +61,7 @@ public void startManagedService(String managedServiceName) {
             throw new IllegalArgumentException("Service:" + managedServiceName + "not registered.");
         }
 
-        Optional<AtomBuilder> optionalAtomBuilder = createBuiler(managedServiceName);
+        Optional<AtomBuilder> optionalAtomBuilder = createBuilder(managedServiceName);
```

Comment:
This typo fix is not related to logging. I have no problem to include in this PR, but the PR name could be "confusing".